R/pkg/R/sparkR.R

```diff
@@ -362,6 +362,10 @@ sparkR.session <- function(
   enableHiveSupport = TRUE,
   ...) {
 
+  if (length(sparkConfig[["spark.sql.warehouse.dir"]]) == 0) {
+    sparkConfig[["spark.sql.warehouse.dir"]] <- tempdir()
```

Comment:
How about an argument named `sparkWorkingDirectory` that defaulted to `tempdir()`?